src/js/app.js

```diff
@@ -1 +1,41 @@
 console.log("test gulp");
+
+$( document ).ready(function() {
+
+  //mask phone
+  jQuery(function($) {
+    $("#phone").mask("+375 (99) 999-99-99");
+  });
```

Comment:
![image](https://user-images.githubusercontent.com/36702984/40681628-572c78a6-6392-11e8-8bd6-e007a1786454.png)
fix this please